pkg/fs/realfs.go

```diff
@@ -43,8 +45,32 @@ func (realFS) Open(name string) (File, error) { return os.Open(name) }
 // Stat delegates to os.Stat.
 func (realFS) Stat(name string) (os.FileInfo, error) { return os.Stat(name) }
 
-// ReadFile delegates to ioutil.ReadFile.
-func (realFS) ReadFile(name string) ([]byte, error) { return ioutil.ReadFile(name) }
+var separator = []byte(`
+---
```

Comment:
This is for yaml.
If a user uses `*.json`, it won't work.

Reply:
That's true. I assume they use `*.yaml`. Shall I add checking for it?

Reply:
What is the separator for JSON content?